features/pageobjects/boards.page.js

```diff
@@ -0,0 +1,131 @@
+const { $, browser, expect } = require("@wdio/globals");
+const Page = require("./page");
+
+class BoardsPage extends Page {
+
+  get createButton() {
+    return $('//button[@data-testid="header-create-menu-button"]');
+  }
+  get creatBoardButton() {
+    return $('//button[@data-testid="header-create-board-button"]');
+  }
+  get boardTitleInput() {
+    return $('//input[@data-testid="create-board-title-input"]');
+  }
+  get finalCreatBoardBtn() {
+    return $('//button[@data-testid="create-board-submit-button"]');
+  }
+   get searchInput() {
+    return $('//input[@data-test-id="search-dialog-input"]');
+  }
+  get boardsLink() {
+    return $('//span[@data-testid="BoardIcon"]');
+  }
+  get workspaceNav() {
+    return $('//button[@data-testid="workspace-switcher"]');
+  }
+  get myWorkspaceLink() {
+    return $('//a[@data-testid="workspace-switcher-popover-tile"]');
+  }
+  get myTrelloBoard() {
+    return $('//a[@title="My Trello board"]');
+  }
+  get addListBtn() {
+    return $('//button[@data-testid="list-composer-button"]');
+  }
+  get listTitleInput() {
+    return $('//textarea[@data-testid="list-name-textarea"]');
+  }
+  get submitNewListBtn() {
+    return $('//button[@data-testid="list-composer-add-list-button"]');
+  }
+  get listElement(){
+    return $('//li[@data-testid="list-wrapper"]')
+  }
+  get addCardBtn(){
+    return $('//button[@data-testid="list-add-card-button"]')
+  }
+  get cardTextareaInput(){
+    return $('//textarea[@data-testid="list-card-composer-textarea"]')
+  }
+  get finalAddCardBtn(){
+    return $('//button[@data-testid="list-card-composer-add-card-button"]')
+  }
+  get cardLink(){
+    return $('//a[@data-testid="card-name"]')
+  }
+  get filterBtn(){
+  return $('//button[@data-testid="filter-popover-button"]')
+  }
+  get filterPopover(){
+    return $('//section[header[h2[@title="Filter"]]]')
+  }
+  get checkboxEl(){
+    return $('//label[@data-testid="clickable-checkbox"]')
+  }
+  get matchedCardsText(){
+    return $('//p[contains(text(), "cards match filters")]')
+  }
+  async openCreatMenu() {
+    await this.createButton.click();
+  }
+  async openCreatBoardDropdown() {
+    await this.creatBoardButton.click();
+  }
+  async setBoardTitle(name) {
+    await this.boardTitleInput.setValue(name);
+  }
+  async creatNewBoard() {
+    await this.finalCreatBoardBtn.click();
+  }
```

Comment:
lines 69-80: methods are unnecessary because you wrap the initial functionality of WDIO
You can use those clicks on steps level